frontend/javascript/components/docupload/image-document.vue

```diff
@@ -34,19 +34,17 @@
               class="form-control"
               :placeholder="i18n.documentTitlePlaceholder" />
           </div>
-          <draggable
-            v-model="pages"
-            class="row pages bg-body-secondary"
-            @start="drag = true"
-            @end="drag = false">
-            <image-page
-              v-for="page in pages"
-              :key="page.pageNum"
-              :page="page"
-              :page-count="pages.length"
-              @pageupdated="$emit('pageupdated', { document, ...$event })"
-              @splitpages="splitPages" />
-          </draggable>
+          <!-- akward @update because Vue 2-ish vs 3.2 interop -->
```

Comment:
Upgrading this to vue 3 syntax probably requires more changes downstream, right? 😬

Reply:
This awkwardness happens where Vue2/3 (resp.: options/composition API based) components touch. *Upgrading* our way out of the awkwardness means upgrading the parent component to Vue3/Composition. Which would then shift the 2/3 touching points to other components.. end cascade Vue3 everywhere :)
...I think.
I'm not familiar with mixed Vue2/3 codebases. I gave a mixed/hybrid SFC a try but quickly ran into weird errors and unsupported dealbreakers. It might work in other places. Not sure if/how/when to *upgrade* stuff to Vue3, other than on a "is it new / is it broken now" basis.